source/spark_sql_migrations/release-notes/release-notes.md

```diff
@@ -1,5 +1,43 @@
 # Spark SQL Migrations Release Notes
 
+## Version 1.10.0
+
+- Added `rollback_on_failure` parameter to the `SparkSqlMigrationsConfiguration` class. This parameter allows the user to specify whether the migration should be rolled back if an error occurs during the migration process. The default value is `False`.
+
+### Changes
+
+As the behavior before was `True`, the default value is now `False`. This means that if an error occurs during the migration process, the migration will not be rolled back by default. If the user wants to roll back the migration in case of an error, they can set the `rollback_on_failure` parameter to `True`.
+
+Example:
+
+``` python
+spark_config = SparkSqlMigrationsConfiguration(
+    ...
+    rollback_on_failure=True
+ )
+```
+
+## Version 1.9.4
+
+- Bug fixed for rollback when more than two migration scripts are running at the same time.
+
+## Version 1.9.3
+
+- SparkSqlMigrationsConfiguration no longer defaults to hive_metastore as catalog name, so now it is not an optional property.
+
+### Changes
+
+The `catalog_name` parameter in the `SparkSqlMigrationsConfiguration` class is no longer optional. The default value was `hive_metastore`, but now it is required to be set by the user.
+
+Example:
+
+``` python
+spark_config = SparkSqlMigrationsConfiguration(
+    ...
+    catalog_name="some_catalog_name"
+ )
+```
```

Comment:
Nice you added this as-well 💪 
For another PR: Can this documentation be enforced when bumping the version? Like we have for NuGet packages.

Reply:
I am currently working on it!